modules/installation-configuration-parameters.adoc

```diff
@@ -2642,6 +2672,63 @@ Additional VMware vSphere configuration parameters are described in the followin
         networks:
 |Lists any network in the vCenter instance that contains the virtual IP addresses and DNS records that you configured.
 |String
+ifdef::agent[]
+
+|platform:
+  vsphere:
+    failureDomains:
+      topology:
+        computeCluster:
+|The path to the vSphere compute cluster.
+|String
+
+|platform:
+  vsphere:
+    failureDomains:
+      topology:
+        datacenter:
+|Lists and defines the datacenters where {product-title} virtual machines (VMs) operate.
+The list of datacenters must match the list of datacenters specified in the `vcenters` field.
+|String
+
+|platform:
+  vsphere:
+    failureDomains:
+      topology:
+        datastore:
+|The path to the vSphere datastore that holds virtual machine files, templates, and ISO images.
+[IMPORTANT]
+====
+You can specify the path of any datastore that exists in a datastore cluster.
+By default, Storage vMotion is automatically enabled for a datastore cluster.
+Red{nbsp}Hat does not support Storage vMotion, so you must disable Storage vMotion to avoid data loss issues for your {product-title} cluster.
+
+If you must specify VMs across multiple datastores, use a `datastore` object to specify a failure domain in your cluster's `install-config.yaml` configuration file.
+For more information, see "VMware vSphere region and zone enablement".
+====
+|String
+
+|platform:
+  vsphere:
+    failureDomains:
+      topology:
+        resourcePool:
+|Optional. The absolute path of an existing resource pool where the installation program creates the virtual machines, for example, `/<datacenter_name>/host/<cluster_name>/Resources/<resource_pool_name>/<optional_nested_resource_pool_name>`.
+// Commenting out the line below because it doesn't apply to Agent, but it may be needed when this content is brought into the regular vSphere docs.
+// If you do not specify a value, resources are installed in the root of the cluster, for example `/example_datacenter/host/example_cluster/Resources`.
+|String
+
+|platform:
+  vsphere:
+    failureDomains:
+      topology:
+        folder:
+|Optional. The absolute path of an existing folder where the installation program creates the virtual machines, for example, `/<datacenter_name>/vm/<folder_name>/<subfolder_name>`.
+// Commenting out the line below because it doesn't apply to Agent, but it may be needed when this content is brought into the regular vSphere docs.
+// If you do not provide this value, the installation program creates a top-level folder in the datacenter virtual machine folder that is named with the infrastructure ID.
+If you are providing the infrastructure for the cluster and you do not want to use the default `StorageClass` object, named `thin`, you can omit the `folder` parameter from the `install-config.yaml` file.
```

Comment:
In case of ABI, the user creates the machines, not the installer program. Also, I am unable to create an ISO without providing the folder, so it is a required field. Since it is a required field for ABI, we also need to modify this line "you can omit the `folder` parameter from the `install-config.yaml` file."
cc: @rwsu

Reply:
Sounds good, I modified the first line to say "folder where the user has created the virtual machines". 

Since the last sentence is based on omitting a required parameter, which we can't do, should I get rid of the sentence entirely, or is there a different way to perform the functionality described there?

Reply:
Per conversation on Slack, I'll remove that last sentence entirely and double check with Richard once it's removed.

Reply:
Manoj is correct. folder is not optional and is a required parameter. I think line 2729 should be commented out.